src/components/Toggle/Toggle.scss

```diff
@@ -0,0 +1,87 @@
+$toggle-track-length: rem(46px);
+$toggle-track-height: rem(24px);
+$toggle-thumb-size: rem(20px);
+$disabled-opacity: 0.6;
+$toggle-thumb-displacement: rem(2px);
+
+.Toggle {
+  display: flex;
+  align-items: center;
+  border: 1px solid transparent;
```

Comment:
This `1px` can be moved into a `toggle-border-size` variable and used throughout the component styles.